docs/docs/1.2.0/deployments/multi-zone.md

```diff
@@ -273,7 +268,7 @@ spec:
   enabled: true
 ```
 
-Changing this value to `enabled: false` will allow the user to exclude the zone's `Ingress` from all other zones - and by doing so - preventing traffic from being able to enter the `zone`. 
+Changing this value to `enabled: false` will allow the user to exclude the zone's `Zone Ingress` from all other zones - and by doing so - preventing traffic from being able to enter the `zone`. 
```

Comment:
Think we can rm "zone's" here -> "...allow the user to exclude the `Zone Ingress` from all other zones ...`